src/Formatter/ESLintFormatter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Formatter;
+
+use Symfony\Component\Process\Process;
+
+class ESLintFormatter implements ProcessFormatterInterface
+{
+    public function format(Process $process): string
+    {
+        $stdout = $process->getOutput();
+        $stderr = $process->getErrorOutput();
+
+        return trim($stdout . PHP_EOL . $stderr);
+    }
+
+    public function formatErrorMessage(array $messages, array $suggestions): string
```

Comment:
This part is a bit confusing, since both inputs are acutally strings that are wrapped in a single-indexed array.
Maybe better to simplify it?

Reply:
cool, I wanted to do that too, done 👍